src/scheduler/server_info.c

```diff
@@ -826,11 +825,11 @@ find_alloc_resource(resource *resplist, resdef *def)
  *
  * @par MT-Safe:	no
  */
-resource *
-find_alloc_resource_by_str(resource *resplist, char *name)
+schd_resource *
```

Comment:
return in function header should now say schd_resource. It's the same case in functions below as well